partner_changeset/README.rst

```diff
@@ -0,0 +1,96 @@
+==================
```

Comment:
Hi @guewen Thanks for the contribution!
I will provide some feedback mostly functional and on the general guidelines.
Please follow the OCA README template (first sections are missing):
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Can you provide a summary of the module easy to understand. Current details are very complete but highly technical.